components/form/CheckboxGroup.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useFormContext } from 'react-hook-form';
+import PropTypes from 'prop-types';
+import { Label, ErrorLabel } from '.';
+
+export function Checkbox({ containerClassName, inputClassName, text, name, value }) {
```

Comment:
`text` should be `label` so that it conforms with the Input component. Please note that I didn't mean that you should change label to text, I only said we should use `text` for label.

```suggestion
export function Checkbox({ containerClassName, inputClassName, label, name, value }) {
```